docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -755,24 +755,208 @@ To read and write a value of the Bluetooth descriptor:
 ### Accessing connection's ATT MTU
 <!-- END #11 -->
 
-<!-- TODO #12
-      - write about registerService()
- -->
 ### Registering services
-<!-- END #12 -->
+To register a GATT service in the local server:
+
+1. Define the init data of the service and all of its components:
+
+   ```
+   var descriptorInitData = {
+     uuid: 'abcd'
+     /* Define more attributes, if needed */
+   };
+
+   var characteristicInitData = {
+     uuid: '9876',
+     descriptors: [descriptorInitData]
+     /* Define more attributes, if needed */
+   };
+
+   var includedServiceInitData = {
+     serviceUuid: '5678',
+     isPrimary: true
```

Comment:
is it possible to create primary service which is included in other primary service?

Reply:
Yes, it's possible. As far as I remember, the only significant difference between primary and secondary services is that secondary services cannot include primary ones. Primary services can include both primary and secondary services.